fxa/tools/fxa.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Naming nit: 'fxa.py' is very generic, this could be something like "create_user.py"